pkg/internalapis/cnsvolumeoperationrequest/cnsvolumeoperationrequest.go

```diff
@@ -285,18 +290,19 @@ func (or *operationRequestStore) StoreRequestDetails(
 // from the operationRequestStore.
 func (or *operationRequestStore) DeleteRequestDetails(ctx context.Context, name string) error {
 	log := logger.GetLogger(ctx)
+	csiNamespace := getCSINamespace()
```

Comment:
Can we can call getCSINamespace once and cache it in the memory instead of reading it multiple times from the environment variable.

Reply:
Made the changes